crates/env/src/lib.rs

```diff
@@ -99,3 +99,24 @@ pub use self::{
         NoChainExtension,
     },
 };
+pub use ink_prelude;
+
+/// Appends a formatted string to the `debug_message` buffer which will be:
+///  - Returned to the caller when the contract is invoked via RPC (*not* via an extrinsic)
+///  - Logged as a `debug!` message on the substrate node, which will be printed to the node
+///    console's `stdout` when the log level is set to `debug`.
+#[macro_export]
+macro_rules! debug_print {
+    ($($arg:tt)*) => ($crate::debug_message(&$crate::ink_prelude::format!($($arg)*)));
```

Comment:
```suggestion
    ($($arg:tt)+) => ($crate::debug_message(&$crate::ink_prelude::format!($($arg)*)));
```

Reply:
would prevent usage of `debug_print!()`

Reply:
This is actually already prevented by the inner call to `format!`, which gives a nice error message:
![image](https://user-images.githubusercontent.com/75586/118947581-6439da00-b94f-11eb-8e0c-2d87ebcb891d.png)